cleanlab/dataset.py

```diff
@@ -76,13 +76,14 @@ def rank_classes_by_label_quality(
 
         By default, the DataFrame is ordered by "Label Quality Score", ascending.
     """
+    if multi_label:
+        raise ValueError("rank_classes_by_label_quality is not supported when multi_label is True")
```

Comment:
TODO: can you make this method work for the multi-label setting, in a follow-up PR after we've finalized this one.

The way to make this work for multi-label data would be to internally call: `common_multilabel_issues()` which will give you two of the key columns you need from this `rank_classes_by_label_quality ` dataframe: "Label Issues" and "Inverse Label Issues".  I believe you should be able to figure out how to return the rest of the columns based off of this for a multi-label dataset. 

Let me know if you have questions then, but let's first fully figure out the `common_multilabel_issues` method

Reply:
Added new functions starting from here: https://github.com/cleanlab/cleanlab/blob/361e05284c5bb5151a49aee7e5aa8ff41ad47220/cleanlab/multilabel_classification/dataset.py#L98